test/pool_test.dart

```diff
@@ -737,7 +738,7 @@ void Function() expectNoAsync() {
 /// A matcher for Futures that asserts that they don't complete.
 ///
 /// This should only be called within a [FakeAsync.run] zone.
-Matcher get doesNotComplete => predicate((future) {
+Matcher get doesNotComplete => predicate((Future future) {
```

Comment:
Why do we need this? Should we delete line 742?

Reply:
Why: guessing it's not liking the dynamic invoke
Will remove the next line!